plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events-v2.ts

```diff
@@ -387,7 +387,12 @@ export function addHistoricalEventsExportCapabilityV2(
     }
 
     async function exportHistoricalEvents(payload: ExportHistoricalEventsJobPayload): Promise<void> {
-        status.info('ℹ️', `Running export historical events with config ${pluginConfig} for payload: ${payload}`)
+        status.info(
+            'ℹ️',
+            `Running export historical events with config ${JSON.stringify(pluginConfig)} for payload: ${JSON.stringify(
```

Comment:
The PluginConfig class is pretty big, IIRC we're only interested in the ID, as the rest should not change
```suggestion
            `Running export historical events with config ${pluginConfig.id} for payload: ${JSON.stringify(
```